generated/niscope/niscope/enums.py

```diff
@@ -147,6 +147,12 @@ class CableSenseMode(Enum):
     '''
 
 
+class _CalibrationTypes(Enum):
+    SELF = 1
+    EXTERNAL = 0
+    MANUFACTURE = 2
+
```

Comment:
`src/niscope/metadata/enums.py` is missing from the PR.

Reply:
No changes were made to that file. CalibrationTypes already existed there but was unused by any part of src prior to now.